IPython/core/ultratb.py

```diff
@@ -299,7 +299,10 @@ def getargs(co):
 
 # Monkeypatch inspect to apply our bugfix.
 def with_patch_inspect(f):
-    """decorator for monkeypatching inspect.findsource"""
+    """
+    Deprecated since IPython 6.0
+    decorator for monkeypatching inspect.findsource
+    """
 
```

Comment:
We may want to raise a deprecation warning as well, but that can be done in a subsequent PR.